File: Codes/Bingran Li/String/58. Length of Last Word.java
public class Solution {
    public int lengthOfLastWord(String s) {
        
        if(s.equals(""))
            return 0;
            
        s = s.trim(); // this api can remove space at front and end----> from 1ms to 0ms
        
        char [] myArray = s.toCharArray();
        int count =0;
    
        for(int i=myArray.length-1;i>-1;i--,count++){
            if(myArray[i]==' ')
                break;
        }
        
        return count;
        
    }
}

File: Codes/Yangkun Huang/String/58. Length of Last Word.cpp
int lengthOfLastWord(string s) {
	int count = 0, result = 0;
	for (int i = 0; i<s.length(); i++)
		if (s[i] != ' ')
		{
		count++;
		result = count;
		}
		else
			count = 0;
	return result;
}